apps/vaporgui/VDoubleSpinBox.cpp

```diff
@@ -0,0 +1,32 @@
+#include "VDoubleSpinBox.h"
+
+VDoubleSpinBox::VDoubleSpinBox( double min, double max )
+: VContainer( this )
```

Comment:
The VContainer takes a `QWidget *containee` as it parameter so here it would be containing itself?

Reply:
Fixed.